environment.yml

```diff
@@ -0,0 +1,61 @@
+name: dirac-development
+
+channels:
+  - conda-forge
+
+dependencies:
+  # runtime
+  - python =2.7
+  - pip
+  - certifi
+  - cmreshandler >1.0.0b4
+  - elasticsearch-dsl ~=6.3.1
+  - fts-rest
+  - future
+  - gitpython >=2.1.0
+  - m2crypto # ==0.32
+  - matplotlib >=2.1.0,<3.0
+  - mysql-connector-c  # Included as it's a dependency of MySQL-python which is installed with pip
+  - numpy >=1.10.1,<1.16  # Limit to 1.15 as Python 2 pylint has bugs with numpy 1.16
+  - pexpect >=4.0.1
+  - psutil >=4.2.0
+  - pyasn1 >0.4.1
+  - pyasn1-modules
+  - pygsi
+  - python-json-logger >=0.1.8
+  - pytz >=2015.7
+  - requests >=2.9.1
+  - rst2pdf >=0.93
+  - six >=1.10
+  - sqlalchemy >=1.0.9
+  - subprocess32
+  - stomp.py >=3.1.5
+  - suds-jurko >=0.6
+  # testing and development
+  - autopep8
+  - caniusepython3
+  - codecov
+  - coverage
+  - hypothesis
+  - ipython
+  - mock
+  - pylint >=1.6.5
```

Comment:
You need to add `parametrized` here (see e.g. https://github.com/DIRACGrid/DIRAC/pull/4103/checks?check_run_id=263589384#step:4:23) which was added to `requirements.txt` from rel-v6r22